src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Publish.targets

```diff
@@ -192,6 +192,12 @@ Copyright (c) .NET Foundation. All rights reserved.
         <RelativePath>@(_DebugSymbolsIntermediatePath->'%(Filename)%(Extension)')</RelativePath>
       </ResolvedFileToPublish>
 
+      <!-- Copy the documentation file (.xml), if any -->
+      <ResolvedFileToPublish Include="@(_DebugSymbolsIntermediatePath)"
```

Comment:
The code you are adding is only publishing the app's doc file, not any P2P references.  So TestLibrary's doc file is not getting published because it is a P2P reference.

I mentioned this in the issue, but I'll say it here:

>Do you also expect that P2P references' doc files are published?  What about doc files from NuGet packages?
> I think since documentation files aren't typically used at runtime, they shouldn't be published by default.
> Projects that need this behavior can add their own target that runs during publish and adds the documentation file to the @(ResolvedFileToPublish) item, which will copy the xml file to the publish directory.  It isn't necessarily behavior that the core SDK has to provide.